connector_salesforce/__openerp__.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{'name': 'Salesforce Connector',
+ 'version': '0.1',
+ 'author': 'Camptocamp,Odoo Community Association (OCA)',
+ 'maintainer': 'Camptocamp,Odoo Community Association (OCA)',
+ 'category': 'Connector',
+ 'complexity': "normal",
+ 'depends': ['connector', 'sale', 'product'],
+ 'summary': """Provides core import export interfaces with Salesforce.""",
+ 'data': [
+     'salesforce_backend/view/backend_model_view.xml',
+     'salesforce_account/view/backend_view.xml',
+     'salesforce_contact/view/backend_view.xml',
+     'salesforce_contact/view/res_partner_view.xml',
+     'salesforce_product/view/backend_view.xml',
+     'salesforce_price_book_entry/view/backend_view.xml',
+     'salesforce_opportunity/view/backend_view.xml',
+     'security/ir.model.access.csv',
+     'data/cron.xml',
+ ],
+ 'external_dependencies': {'python': ['simple_salesforce', 'requests']},
+ 'test': [],
+ 'installable': True,
```

Comment:
not sure whether you need to add those keys if the value is the default one (except licence of course)